main.py

```diff
@@ -1021,9 +1027,9 @@ def scenario_plot(location, target, scenario, ui, age_group, round_tab, ens_chec
     Input("sample-slider", "value"),
     Input("median-checkbox", "value"),
     Input("tabs-round", "value"))
-def spaghetti_plot(location, target, scenario, age_group, n_sample, med_plot, round_tab):
+def spaghetti_plot(location, target, scenario, age_group, n_sample, med_plot, round_tab, band_depth_limit=None):
```

Comment:
Proposed documentation to add:
    """
    :param location: str used to construct path to sample file to load in visualization/data-visualization
    :param target: str, 'inc hosp' or 'cum hosp'. Used to construct path to sample file to load in visualization/data-visualization
    :param scenario: List[int], used to filter sample file
    :param age_group: str, used to filter sample file
    :param n_sample: int, number of samples to display
    :param med_plot: bool, if True, median trajectory displayed
    :param round_tab: str, name of Round (as seen in hub-config/viz_settings.json) to be used
    :param band_depth_limit: int| None. If not None, will show an envelope in resulting spaghetti plot around trajectories with a band depth above this value. Band depth is a measure of the representativeness of one trajectory among an ensemble. For more details, see https://ieeexplore.ieee.org/document/6875964 - Curve Boxplot: Generalization of Boxplot for Ensembles of Curves by Mirzargar et al.
    :return: go.Figure object result
    """